app/templates/components/explore-side-bar.hbs

```diff
@@ -0,0 +1,109 @@
+{{#ui-sticky}}
```

Comment:
@niranjan94 Sorry, I am a little confused with 

> for now, let everything be loaded

What do you mean by loaded